portfolio.js

```diff
@@ -2,15 +2,81 @@ const appearMenu = document.querySelector('.menuDropDown');
 const mobileMenu = document.querySelector('.default');
 const disappearMenu = document.querySelector('.x-icon');
 const disappearMenuOptions = document.querySelectorAll('.mobile-menu-li');
+const popupClick = document.querySelectorAll('.btn-Project');
 
+const portfolio = [
+  {
+    img: 'images/Snapshoot-Portfolio1.svg',
+    name: 'Tonic',
+    information1: 'CANOPY',
+    information2: 'Back End Dev',
+    information3: '2015',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technolgies1: 'html',
+    technologies2: 'css',
+    technologies3: 'javascript',
+    seeLive: 'https://oshanedesign.github.io/My-Portfolio/',
+    seeSource: 'https://github.com/oshanedesign/My-Portfolio',
+    span1: 'See live',
+    span2: 'See source',
+  },
+];
```

Comment:
- [ ] Kindly here you should add data for every project in your page, in your template you have 4 projects so you need to have 4 objects and when click on specific project its data show up in the modal.
```
[
{ project one object ...},
{ project two object ...},
{ project three object ...},
{ project four object ...}
]
```